do.py

```diff
@@ -459,6 +460,42 @@ def getstatusoutput(command):
     )
 
 
+def build(clean_env="false", sdk="all"):
+    print("\nStep 1: Set up virtaul env, if does not exsists already")
```

Comment:
remove `if does not *` since you're already printing the messages below

Reply:
done